test/extension/chromeMock.js

```diff
@@ -13,6 +13,35 @@ import { readFile } from '@web/test-runner-commands';
 
 const ID = 'dummy';
 
+class StorageMock {
+  constructor(state = {}) {
+    this.state = state;
+  }
+
+  get(name, callback) {
+    const ret = {};
+    ret[name] = this.state[name];
+    callback(ret);
```

Comment:
```suggestion
    callback({
      [name]: this.state[name],
    });
```

Reply:
I _knew_ there was a more elegant way to write this - thanks!